engine/src/main/java/org/terasology/rendering/assets/shader/ShaderProgramFeature.java

```diff
@@ -21,11 +21,11 @@
 /**
  */
 public enum ShaderProgramFeature {
-    FEATURE_REFRACTIVE_PASS (0b00000001),
-    FEATURE_ALPHA_REJECT (0b00000010),
-    FEATURE_LIGHT_POINT (0b00000100),
-    FEATURE_LIGHT_DIRECTIONAL (0b00001000),
-    FEATURE_USE_MATRIX_STACK (0b00100000),
+    FEATURE_REFRACTIVE_PASS      (0b00000001),
+    FEATURE_ALPHA_REJECT         (0b00000010),
+    FEATURE_LIGHT_POINT          (0b00000100),
+    FEATURE_LIGHT_DIRECTIONAL    (0b00001000),
+    FEATURE_USE_MATRIX_STACK     (0b00100000),
```

Comment:
I actually commented earlier, but it sounds like in the time it took me to review and comment this change was made and my comment got swallowed as having been made against an obsolete commit :-)

Historically we've avoided space-lining up things as it is hard to enforce consistency and maybe even impossible to code rules for it (one tool may warn about the extra whitespace while another ignores it). It may also end up disrupted again later.

Fundamentally I don't mind and we can leave it in, it just may trigger other whitespace style warnings somewhere.